tests/unit/store/mutations.spec.ts

```diff
@@ -5,10 +5,52 @@ import StuntSheet from "@/models/StuntSheet";
 import ContInPlace from "@/models/continuity/ContInPlace";
 import ContETFDynamic from "@/models/continuity/ContETFDynamic";
 import ContEven from "@/models/continuity/ContEven";
+import StuntSheetDot from "@/models/StuntSheetDot";
 
 describe("store/mutations", () => {
   let store: Store<CalChartState>;
 
+  describe("syncDotLabelIndices", () => {
+    // TODO: Update and add tests upon updating syncDotLabelIndices
```

Comment:
Do we have an issue for this? TODO comments are good for communicating to a reader that something is unfinished, but unless we throw it into the issue tracker, we might lose track of this.